File: test/fixtures/ruby-httparty/get_user_agent_alias.rb
require 'httparty'

url = 'http://localhost:28139/vc/moviesmagic?p=5&pub=testmovie&tkn=817263812'
headers = {
  'x-msisdn': 'XXXXXXXXXXXXX',
  'user-agent': 'Mozilla Android6.1',
}
res = HTTParty.get(url, headers: headers)

File: test/fixtures/node-got/get_user_agent_alias.js
import got from 'got';

const response = await got('http://localhost:28139/vc/moviesmagic', {
  searchParams: {
    'p': '5',
    'pub': 'testmovie',
    'tkn': '817263812'
  },
  headers: {
    'x-msisdn': 'XXXXXXXXXXXXX',
    'user-agent': 'Mozilla Android6.1'
  }
});
